atomic_reactor/plugins/pre_check_and_set_platforms.py

```diff
@@ -64,4 +65,17 @@ def run(self):
         else:
             platforms = get_orchestrator_platforms(self.workflow)
 
-        return get_platforms_in_limits(self.workflow, platforms)
+        if not platforms:
+            # Not an orchestrated build
```

Comment:
If koji target is used, this won't be None. I think this comment is misleading. We probably need different heuristics for determining whether or not this is an orchestrator build. `self.reactor_config.is_default()` could be used to determine if there is a reactor config at all which seems like a safer check.

Reply:
Well, yes, the comment is misleading because this plugin is only ever used for orchestrator builds (assuming that plugin configurations generated by osbs-client are the only valid ones!), so if this gets hit, it basically means that an orchestrator build was triggered without platforms and without a koji_target - which will cause a failed build. It probably should just raise a RuntimeError() here rather than waiting for OrchestratorBuildPlugin.run()

I propagated the comment after writing it in util.get_platforms() where a None return does imply 'not an orchestrat**or** build'. (So a prod or worker build.) I've looked through the plugins that use get_platforms(), and they seem to have reasonable looking handling for a None return from get_platforms().

I've added the RuntimeError here and changed orchestrated to orchestrator in get_platforms() in the version I'm about to push.